Mentioning the limitations of YAML files in the docs (#1668)
terrapower/armi: doc/user/inputs.rst

  	activate) and all kind of modeling approximation settings (e.g. convergence criteria)

Blueprints file
	Contains dimensions and composition of the components/blocks/assemblies in your reactor systems,

>     from fuel pins to heat exchangers

Reviewer: I expect this might need to be indented.

Author: There were some indenting problems in this file (spaces mixed with tabs), I fixed them.

 
Fuel management file
	Describes how fuel moves around during a simulation

Depending on the type of analysis, developers may create other input files for things like: control
logic, ex-core models for transients and shielding, etc.

YAML Files
==========
ARMI's input files all use the `YAML <https://en.wikipedia.org/wiki/YAML>`_ format. This is a well-
known file format, chosen because it is human-readable and easy to hand-write. That being said,

> there are two details about the YAML format it is important to know:

Reviewer: ```suggestion
there are two details about the YAML format that are important to know:
```

Author: Fixed
